lndr-backend/src/Lndr/Types.hs

```diff
@@ -170,6 +176,7 @@ $(deriveJSON (defaultOptions { fieldLabelModifier = over _head toLower . drop 12
 data NotificationAction = NewPendingCredit
                         | CreditConfirmation
                         | PendingCreditRejection
+                        | Blank
```

Comment:
There is not need to have a blank notification. Just don't call `attemptToNotify` if there's nothing to say.